sources/api/certdog/src/main.rs

```diff
@@ -54,34 +57,34 @@ struct CertBundle {
 
 /// Query the API for the certificate bundles, returns a tuple with trusted
 /// and distrusted PEM certificates
-async fn get_certificate_bundles<P>(socket_path: P) -> Result<CertBundle>
+async fn get_certificate_bundles<P>(config_path: P) -> Result<CertBundle>
 where
     P: AsRef<Path>,
 {
-    debug!("Querying the API for settings");
-
-    let method = "GET";
-    let uri = constants::API_SETTINGS_URI;
-    let (_code, response_body) = apiclient::raw_request(&socket_path, uri, method, None)
-        .await
-        .context(error::APIRequestSnafu { method, uri })?;
+    debug!("Loading certdog configuration");
+    if !config_path.as_ref().exists() {
+        return split_bundles(None);
```

Comment:
Seems strange to announce the loading of the configuration if it can be skipped in the next line. (I don't know if this is worthy of a warning or if it is fine. If it's a problem should this actually be an error condition instead?)

```suggestion
        warn!("Configuration file does not exist, skipping")
        return split_bundles(None);
```